views/courses/components/template-new-course-time.hbs

```diff
@@ -1,6 +1,7 @@
 <!-- just for copying, add input names programmatically! -->
-<tr class="new-course-time-template">
-    <td><button type="button" class="course-time-delete" aria-label="{{$t "global.headline.delete"}}" data-id="{{this._id}}">
+<tr class="new-course-time-template" style="vertical-align: top;">
```

Comment:
why you put style property here instead of to the class?

Reply:
done